src/schema/v2/schema.ts

```diff
@@ -501,6 +502,7 @@ export default new GraphQLSchema({
       matchArtworkImportArtists: MatchArtworkImportArtistsMutation,
       matchArtworkImportRowImage: MatchArtworkImportRowImageMutation,
       mergeArtists: mergeArtistsMutation,
+      editMeOrder: editMeOrderMutation,
```

Comment:
Try to keep these alpha

Reply:
alphabetized the whole list in https://github.com/artsy/metaphysics/pull/6506/commits/c7bd7b6f9b3ff56796dd2abbf3eb8b22e362c2bc